src/coreclr/minipal/Unix/dn-u16.cpp

```diff
@@ -5,13 +5,11 @@ typedef char16_t WCHAR;
 
 #include <dn-u16.h>
 #include <string.h>
+#include <minipal/strings.h>
 
 size_t u16_strlen(const WCHAR* str)
 {
-    size_t nChar = 0;
-    while (*str++)
-        nChar++;
-    return nChar;
+    return minipal_u16_strlen((CHAR16_T*)str);
```

Comment:
These can be replaced too:
https://github.com/dotnet/runtime/blob/e377401c233637e761d597115ab42b1e23b757b4/src/mono/mono/eglib/giconv.c#L308
https://github.com/dotnet/runtime/blob/e377401c233637e761d597115ab42b1e23b757b4/src/mono/mono/eglib/giconv.c#L353
https://github.com/dotnet/runtime/blob/e377401c233637e761d597115ab42b1e23b757b4/src/mono/mono/eglib/giconv.c#L395